README.md

```diff
@@ -1 +1 @@
-# Sea-Foods
\ No newline at end of file
+# Sea Foods
```

Comment:
I would suggest you add other sections (e.g authors, Built with e.t.c) to your README.md file